tests/rds_utils/test_db_ingest_query.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+from sds_data_manager.lambda_code.SDSCode.rds_utils.db_ingest_query import DbIngestQuery
+
+
+@pytest.fixture()
+def create_db_query():
+    file_path = "/fake/file/path/file.pkts"
+    metadata = {
+        "mission": "imap",
+        "instrument": "swe",
+        "level": "l1a",
+        "date": "20231030",
+    }
+    return DbIngestQuery(file_path, metadata)
+
+
+def test_db_ingest_init(create_db_query):
+    """Test that DBIngestQuery object correctly initializes"""
+    ## Arrange
+    query_true = """
```

Comment:
Minor nit, but I find the `true` to be confusing in this context, I see `expected` most of the time.